controllers/namespacescope_controller.go

```diff
@@ -128,6 +128,20 @@ func (r *NamespaceScopeReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 		return ctrl.Result{}, err
 	}
 
+	for _, namespaceMember := range instance.Spec.NamespaceMembers {
+		if rolesList, _ := r.GetRolesFromNamespace(instance, namespaceMember); rolesList != nil {
+			var summarizedRules []rbacv1.PolicyRule
+			for _, role := range rolesList.Items {
+				summarizedRules = append(summarizedRules, []rbacv1.PolicyRule(role.Rules)...)
+			}
+
+			if err := r.CreateRuntimeRoleToNamespace(instance, summarizedRules); err != nil {
+				klog.Errorf("Failed to create runtime role: %v", err)
+				return ctrl.Result{}, err
```

Comment:
@ZhuoxiLi We don't expect the Role creating failure will break or impact the rest of the functions. Could we down-level the error messages from error to warning and don't return err reconciliation result?